### PR TITLE
feat: Add export as pentabarfxml

### LINKED ISSUE #4880: Add export event as pentabarf XML
**Is your feature request related to a problem? Please describe.**
Exporting event as pentabarf XML hasn't been implemented yet on the server. It is present in version 1.

**Describe the solution you'd like**
Follow the method used in version 1.

**Describe alternatives you've considered**
\-

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #4880 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Add feature to export event as pentabarf XML

#### Changes proposed in this pull request:
- Followed steps of version 1

### CODE REVIEW

File: app/api/helpers/pentabarfxml.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+class PentabarfExporter:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def export(event_id):
+        """Takes an event id and returns the event in pentabarf XML format"""
+        event = EventModel.query.get(event_id)
```
Comment (srv-twry) [inline app/api/helpers/pentabarfxml.py]: What if the event doesn't exist i.e. a wrong event_id is sent ?

Comment (mayank8318) [reply app/api/helpers/pentabarfxml.py]: @srv-twry Thanks for pointing that out. Using safe_query in the celery task method now. Please review :)

**Comment by srv-twry** [comment]
Also please check if you're using safe query in other export tasks as well. If you aren't then please create an issue and fix it :)

**Comment by mayank8318** [comment]
@srv-twry Yes made that change for other tasks as well. I will look into the failing test now

File: tests/hook_main.py [hunk 2]
```diff
@@ -3428,6 +3428,44 @@ def event_export_post(transaction):
...
+    :param transaction:
+    :return:
+    """
+    with stash['app'].app_context():
+        event = EventFactoryBasic()
+        db.session.add(event)
+        db.session.commit()
+
+
+@hooks.before(
```
[the remark this answers is not in the document]
Comment (mayank8318) [reply tests/hook_main.py]: Resolved

File: tests/hook_main.py [hunk 3]
```diff
@@ -3428,6 +3428,44 @@ def event_export_post(transaction):
...
```
[the remark this answers is not in the document]
Comment (mayank8318) [reply tests/hook_main.py]: Resolved

File: tests/hook_main.py [hunk 4]
```diff
@@ -3428,6 +3428,44 @@ def event_export_post(transaction):
         db.session.commit()
 
 
+@hooks.before(
```
[the remark this answers is not in the document]
Comment (mayank8318) [reply tests/hook_main.py]: Resolved

**Pull request merged**